pom.xml

```diff
@@ -154,17 +181,25 @@
           <groupId>org.glassfish.jersey.media</groupId>
           <artifactId>jersey-media-json-jettison</artifactId>
         </exclusion>
+        <!-- Provided by Jenkins core -->
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
         <exclusion>
           <groupId>org.springframework</groupId>
           <artifactId>spring-beans</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.springframework</groupId>
+          <artifactId>spring-core</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.springframework</groupId>
+          <artifactId>spring-jcl</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.12.0</version>
-    </dependency>
```

Comment:
`commons-io` is already on the classpath by virtue of the dependency of Jenkins core. It makes no sense to manage our own version here or to bundle Commons IO in the HPI, because core's copy will always be loaded by the Jenkins plugin class loading system.